test/access_control.sh

```diff
@@ -33,6 +27,26 @@ function check_es_acls() {
 }
 
 function cleanup() {
+    local return_code="$?"
+    cleanup_es
+    set +e
+	if [ "${DO_CLEANUP:-true}" == "true" ] ; then
+		mkdir -p $ARTIFACT_DIR/$test_name
+		oc -n $LOGGING_NS get configmap elasticsearch -o jsonpath={.data} --ignore-not-found > $ARTIFACT_DIR/$test_name/elasticsearch-configmap.log ||:
+		get_all_logging_pod_logs $ARTIFACT_DIR/$test_name
+
+		os::log::info "Removing test namespaces...."
+		oc delete ns/openshift-logging ns/openshift-operators-redhat --force --grace-period=0
+		for item in "ns/openshift-logging" "ns/openshift-operators-redhat"; do
+			os::cmd::try_until_failure "oc get ${item}" "$((1 * $minute))"
+		done
+	fi
+    exit $return_code
+}
+
+
+function cleanup_es() {
```

Comment:
Do we really need this step?  Most of our tests are moving to simply removing ES entirely and creating anew

Reply:
this helps to save time when i test locally.